<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemLogPage</class>
 <widget class="QWidget" name="SystemLogPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>1272</width>
     <height>712</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTreeView" name="eventLogView">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>4</y>
      <width>1264</width>
      <height>522</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <property name="autoScrollMargin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPageDn">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>620</y>
      <width>120</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPageUp">
    <property name="geometry">
     <rect>
      <x>1059</x>
      <y>620</y>
      <width>120</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPage">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>530</y>
      <width>239</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>88888/88888</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSearch">
    <property name="geometry">
     <rect>
      <x>521</x>
      <y>620</y>
      <width>330</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPrevious">
    <property name="geometry">
     <rect>
      <x>1178</x>
      <y>530</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonClose">
    <property name="geometry">
     <rect>
      <x>1178</x>
      <y>620</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonFunction">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>620</y>
      <width>91</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="searchStartTime">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>530</y>
      <width>430</width>
      <height>90</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="searchEndTime">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>620</y>
      <width>430</width>
      <height>90</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelStart">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>530</y>
      <width>87</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelEnd">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>620</y>
      <width>87</width>
      <height>90</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>End</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFilter1">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>530</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Filter
Sort</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPrevious">
    <property name="geometry">
     <rect>
      <x>1178</x>
      <y>530</y>
      <width>90</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelSerarch">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>620</y>
      <width>331</width>
      <height>90</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelFunction">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>620</y>
      <width>91</width>
      <height>90</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelPageDn">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>620</y>
      <width>120</width>
      <height>90</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelPageUp">
    <property name="geometry">
     <rect>
      <x>1059</x>
      <y>620</y>
      <width>120</width>
      <height>90</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelClose">
    <property name="geometry">
     <rect>
      <x>1178</x>
      <y>620</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelFilter4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>530</y>
      <width>421</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelFilter3">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>530</y>
      <width>30</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>:
:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFilter2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>530</y>
      <width>301</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>searchStartTime</zorder>
   <zorder>searchEndTime</zorder>
   <zorder>labelClose</zorder>
   <zorder>labelPageDn</zorder>
   <zorder>labelPageUp</zorder>
   <zorder>labelFunction</zorder>
   <zorder>buttonFunction</zorder>
   <zorder>labelSerarch</zorder>
   <zorder>labelPrevious</zorder>
   <zorder>eventLogView</zorder>
   <zorder>buttonPageDn</zorder>
   <zorder>buttonPageUp</zorder>
   <zorder>labelPage</zorder>
   <zorder>buttonSearch</zorder>
   <zorder>buttonPrevious</zorder>
   <zorder>buttonClose</zorder>
   <zorder>labelStart</zorder>
   <zorder>labelEnd</zorder>
   <zorder>labelFilter1</zorder>
   <zorder>labelFilter3</zorder>
   <zorder>labelFilter2</zorder>
   <zorder>labelFilter4</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
